0007-configuration-management-extension/README.md

```diff
@@ -39,130 +44,148 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
-## Config resource
-Config resource is used for distributing the latest configuration to endpoints.
+<!-- TODO: move observe extension to 1/KP -->
+7/CMP follows client-initiated request/response pattern defined in [1/KP](/0001-kaa-protocol/#requestresponse-pattern) with observe extension.
+For MQTT, this is achieved by publishing multiple responses to the response topic; for CoAP, it is achieved with the `Observe` option defined in [RFC 7641](https://tools.ietf.org/html/rfc7641).
 
-The config resource is a request/response resource with an observe extension which resides at the following resource path:
+
+
+## Configuration resource
+
+Configuration resource is used by endpoints for subscribing and receiving the latest configuration from server.
+
+The configuration resource is a request/response resource with an observe extension which resides at the following extension-specific resource path:
 ```
-/<endpoint_token>/config
+/<endpoint_token>/config/json
 ```
 
-### Request
-Request payload MUST be a UTF-8 encoded object with the JSON Scheme as defined in [config-request.schema.json](./config-request.schema.json) file.
+### Configuration resource request
+
+The client SHOULD send configuration resource requests to the configuration resource path.
+
+The request payload MUST be a UTF-8 encoded JSON object with the following [JSON schema](http://json-schema.org/) ([config-request.schema.json](./config-request.schema.json)):
 
 ```json
 {
-  "$schema": "http://json-schema.org/schema#",
-  "title": "7/CMX config request schema",
-
-  "type": "object",
-  "properties": {
-    "configId": {
-      "type": "string",
-      "description": "Identifier of the currently applied configuration."
+    "$schema":"http://json-schema.org/schema#",
+    "title":"7/CMP configuration request schema",
+
+    "type":"object",
+    "properties":{
+        "configId":{
+            "type":"string",
+            "description":"Identifier of the currently applied configuration"
+        },
+        "observe":{
+            "type":"boolean",
+            "description":"Whether the endpoint is interested in observing its configuration"
+        }
     },
-    "observe": {
-      "type": "boolean",
-      "description": "Whether the endpoint is interested in observing its configuration."
-    }
-  },
-  "additionalProperties": false
+    "additionalProperties":false
 }
 ```
 
-If `configId` field is missing, the server MUST respond with the current configuration for the given endpoint.
+If `configId` field is missing, server MUST respond with the current configuration for the given endpoint.
 
-If `configId` field is present, the server MUST send new configuration only if `configId` differs from the identifier of the currently assigned cofiguration for the endpoint.
+If `configId` field is present, server MUST send new configuration only if `configId` differs from the identifier of the current endpoint configuration.
```

Comment:
"current endpoint configuration" may refer to both configurations assigned to the endpoint by the server and one currently applied by the endpoint. Previous wording was unambiguous.